exporters/variables-scss/src/index.ts

```diff
@@ -46,35 +48,34 @@ Pulsar.export(async (sdk: Supernova, context: PulsarContext): Promise<Array<AnyO
     return createTextFile(file.path, file.fileName, file.content);
   });
 
-  // TODO: Only for debugging purposes, remove for production!
-  const safeStringify = (obj: object) => {
-    let cache: string[] | null = [];
-    const str = JSON.stringify(
-      obj,
-      (key, value) => {
-        if (typeof value === 'object' && value !== null) {
-          if (cache?.includes(value)) {
-            return 'CIRCULAR_REFERENCE';
+  // Export the original data only if 'generateOriginalDataFiles' is set to true in config.local.json
+  if (exportConfiguration.generateOriginalDataFiles) {
+    const safeStringify = (obj: object) => {
```

Comment:
IMHO, I would suggest moving this function to something like an `utils.ts/helpers.ts` and creating a test for it :-)

Reply:
good point - moved function to separate file and created test for it